src/printer.js

```diff
@@ -1110,7 +1110,16 @@ function genericPrintNoParens(path, options, print) {
             n.expression.type === "LogicalExpression");
 
       if (shouldInline) {
-        return concat(["{", path.call(print, "expression"), "}"]);
+        const shouldPutBracketOnNextLine =
```

Comment:
Technically, `CallExpression` could if there's a member expression chain that ends up indenting everything, but it's very hard to figure out if that's going to happen or not :x